package.json

```diff
@@ -97,6 +97,26 @@
           }
         ],
         "priority": "option"
+      },
+      {
+        "viewType": "one.viewer.metadata",
+        "displayName": "Metadata",
+        "selector": [
+          {
+            "filenamePattern": "*"
```

Comment:
Could you specify the target filename patterns?

Reply:
There are five types of files that we need to set. But I don't know how to set up multiple files. It is not allowed to be treated in an array. Therefore, all file patterns were given as "priority": "option" and processed so that the viewer could only be opened in a certain file.
("when": "view == OneExplorerView && viewItem =~ /baseModel|product/")